src/CommunityToolkit.Maui/Views/Snackbar/Helpers/NativeRoundedStackView.ios.macos.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using CoreAnimation;
+using CoreGraphics;
+using UIKit;
+
+namespace CommunityToolkit.Maui.UI.Views.Snackbar.Helpers
+{
+	class NativeRoundedStackView : UIStackView
+	{
+		public nfloat Left { get; }
+
+		public nfloat Top { get; }
+
+		public nfloat Right { get; }
+
+		public nfloat Bottom { get; }
+
+		public NativeRoundedStackView(nfloat left, nfloat top, nfloat right, nfloat bottom)
+		{
+			Left = left;
+			Top = top;
+			Right = right;
+			Bottom = bottom;
+		}
+
+		public override void Draw(CGRect rect)
+		{
+			ClipsToBounds = true;
+			var path = GetRoundedPath(rect, Left, Top, Right, Bottom);
+			var maskLayer = new CAShapeLayer { Frame = rect, Path = path };
+			Layer!.Mask = maskLayer;
```

Comment:
Let's avoid using the Null Forgiving Operator, `!.`. 

It opens the door to future bugs in future code changes.